From 32868be745da42de977af1a838e19239a5d8ae4f Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Tue, 6 Aug 2019 17:00:24 +0800
Subject: [PATCH] Add WRLinux1019 oval check for rule
 package_rsh-server_removed

Add WRLinux1019 specific oval check for rule package_rsh-server_removed

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 .../package_rsh-server_removed/bash/wrlinux1019.sh |  3 +++
 .../oval/wrlinux1019.xml                           | 26 ++++++++++++++++++++++
 2 files changed, 29 insertions(+)
 create mode 100644 linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/bash/wrlinux1019.sh
 create mode 100644 linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/oval/wrlinux1019.xml

diff --git a/linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/bash/wrlinux1019.sh b/linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/bash/wrlinux1019.sh
new file mode 100644
index 0000000..5a198b9
--- /dev/null
+++ b/linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/bash/wrlinux1019.sh
@@ -0,0 +1,3 @@
+# platform = multi_platform_wrlinux
+
+rpm -e --nodeps rsh-server
diff --git a/linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/oval/wrlinux1019.xml b/linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/oval/wrlinux1019.xml
new file mode 100644
index 0000000..86d4416
--- /dev/null
+++ b/linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/oval/wrlinux1019.xml
@@ -0,0 +1,26 @@
+<def-group>
+ <!-- THIS FILE IS GENERATED by create_package_removed.py.  DO NOT EDIT.  -->
+  <definition class="compliance" id="package_rsh-server_removed"
+  version="1">
+    <metadata>
+      <title>Package rsh-server Removed</title>
+      <affected family="unix">
+        <platform>multi_platform_wrlinux</platform>
+      </affected>
+      <description>The RPM package rsh-server should be removed.</description>
+      <reference source="swells" ref_id="20130829" ref_url="test_attestation"/>
+    </metadata>
+    <criteria>
+      <criterion comment="package rsh-server is removed"
+      test_ref="test_package_rsh-server_removed" />
+    </criteria>
+  </definition>
+  <linux:rpminfo_test check="all" check_existence="none_exist"
+  id="test_package_rsh-server_removed" version="1"
+  comment="package rsh-server is removed">
+    <linux:object object_ref="obj_package_rsh-server_removed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_package_rsh-server_removed" version="1">
+    <linux:name>rsh-server</linux:name>
+  </linux:rpminfo_object>
+</def-group>
-- 
1.9.1

